extension/manifest.json

```diff
@@ -25,6 +25,14 @@
         "48": "images/GearHeart_Mixed_47x47.png",
         "128": "images/GearHeart_Mixed_127x127.png"
     },
+    "commands": {
+        "_execute_browser_action": {
+            "suggested_key": {
+                "windows": "Ctrl+Shift+U",
+                "mac": "Command+Shift+U"
```

Comment:
The way it is currently configured, what would happen Linux? I think you should just set what is "windows" to be "default". I guess you can leave "mac" to be explicit and prevent any issues if "Ctrl" stops automatically mapping to "Command" on Mac.

Reply:
I thought we have to configure only for windows and mac. I'm changing it!